test/parallel/test-tls-wrap-timeout.js

```diff
@@ -27,7 +27,9 @@ var server = tls.createServer(options, function(c) {
 
 server.listen(common.PORT, function() {
   var socket = net.connect(common.PORT, function() {
-    socket.setTimeout(240, assert.fail);
+    socket.setTimeout(common.platformTimeout(240), function () {
```

Comment:
The test did show an error `undefined undefined undefined`, which I thought looked ugly. If that's prefered, I'll undo it.